packages/layout/header.jade

```diff
@@ -1,2 +1,26 @@
-template(name='header')
-  p HEADER
\ No newline at end of file
+template(name="header")
+  header
+    nav.navbar
+      .fluid-container
+        .container
+          .navbar-header
+            button.navbar-toggle.collapsed(type="button", data-toggle="collapse" data-target=".navbar-collapse") Menu
+            a.navbar-brand(href="/")
+              span Mantle
+          .navbar-collapse.collapse
+            ul.nav.navbar-nav.navbar-right
+              +navLinks
+
+template(name="navLinks")
+  if currentUser
+    li.nav-icon.dropdown
+      a.dropdown-toggle.admin-settings(href="#" data-toggle="dropdown" aria-expanded="false")
+        i.fa.fa-gear
+        span Settings
+      ul.dropdown-menu(role="menu")
+        +accountsHeaderButtons
+  else
+    li
+      a(href="/signIn" class="sign-in navbar-btn")
+        span Sign In 
+        i.fa.fa-sign-in
```

Comment:
The sign-in link should part of the accounts header buttons template or at least the accounts package. I don't think we need the settings dropdown at this point, we haven't decided anything about the mantle header yet.